docs/source/DeveloperDocumentation/TaskEngineWorkflowFeatures.md

```diff
@@ -391,3 +391,67 @@ The AVOD example below shows three clips being stitched together. The first clip
   }
 }
 ```
+
+```json
+
+```
+
+## LIVE COMPOSE WITH MANIFEST MANIPULATION
+
+The Vualto Task Engine now supports manifest manipulation to generate live compose streams (looping of a VOD sources' playlist), using AWS Mediatailor Channel Assembly. This engine takes VOD streaming URLs directly and the resulting Live stream fragments come frome the original VOD streaming URLs - this could result in cost savings in caching layers in some instances.
```

Comment:
`looping of a VOD sources' playlist` - this bit is a bit funny to read. Maybe something like `VOD playlist looping` is easier to read and understand?